weblate/static/styles/screenshots/screenshot_detail.css

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © Michal Čihař <michal@weblate.org>
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+#sources-listing {
```

Comment:
I think it would look nicer if only the inner table was scrollable, not the full panel.